lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -1283,7 +1283,6 @@ class DefaultAssetPickerBuilderDelegate
             currentIndex: _index,
             previewAssets: _current,
             themeData: theme,
-            previewThumbSize: previewThumbSize,
```

Comment:
Nope. The fact that HEIC can displays is we use thumbnails for them.

Reply:
Thumbnails always display faster than the original without doubt. Thumbnails in the size of the device will provide previews in the quality between ultra and high since that's almost all the information(pixels) users might need when they looking at the preview.

Reply:
If we want to display pixels as original as possible, we should implement some features like progressive loading when the user is scaling.

Reply:
(e.g. https://jmperezperez.com/medium-image-progressive-loading-placeholder/)

Reply:
I'm not sure what you are suggesting here.
If we allow reduced resolution of the thumbnails for the grid, that resolution might not be good enough for the preview. agree?
So maybe we should use the default resolution (300x300) for the preview, and for the grid the user can specify his own custom resolution if he wants.
(Currently the resolution supplied by the user has no effect on the grid thumbs)